plugins/modules/api.py

```diff
@@ -312,7 +313,7 @@ def __init__(self):
                                         self.module.params['ca_path'],
                                         )
 
-        self.path = self.list_remove_empty(self.module.params['path'].split(' '))
+        self.path = self.list_remove_empty(self.split_params(self.module.params['path']))
```

Comment:
I'm not sure, but shouldn't the path split be split by spaces? I.e.
```suggestion
        self.path = self.module.params['path'].split()
```
After all `'  a b   c'.split() == ['a', 'b', 'c']`, and the path cannot contain any escaping or strings AFAIK?

Reply:
@NikolayDachev what do you think?